variable.go

```diff
@@ -127,6 +127,16 @@ func (m *Variable) Valid() error {
 		return fmt.Errorf("invalid arguments type")
 	}
 
+	inValidNames := [11]string{"and", "import", "not", "return", "option", "test", "empty", "in", "or", "package", "builtin"}
+
+	for x := 0; x < len(inValidNames); x++ {
```

Comment:
While this is valid, it's not preferred.
```suggestion
	for x := range inValidNames {
```

Reply:
got it, i changed it